cmd/kubermatic-installer/cmd_mirror_images.go

```diff
@@ -85,6 +89,12 @@ func MirrorImagesCommand(logger *logrus.Logger, versions kubermaticversion.Versi
 			if len(args) >= 1 {
 				opt.Registry = args[0]
 			}
+
+			if strings.Contains(opt.Registry, "local://") {
```

Comment:
this should be `strings.HasPrefix`, since otherwise `test.test.local://registry.com` would be matched as well. That's of course not a valid registry name, but I think we should make sure the `local://` prefix only exists where we expect it to be.